README.md

```diff
@@ -246,8 +264,25 @@ git restore --merge FILE1 FILE2 FILE3
 
 to return back to the unmerged state.
 
+#### `nix fmt` (experimental)
+
+[nix fmt](https://nix.dev/manual/nix/latest/command-ref/new-cli/nix3-fmt) (part of the [`flakes` experimental feature](https://nix.dev/manual/nix/latest/development/experimental-features#xp-feature-flakes)) can be configured to use `nixfmt` by setting the `formatter` flake output to the respective package (assuming a `nixpkgs` flake input exists):
+
+```nix
+# flake.nix
+{
+  outputs =
+    { nixpkgs, self }:
+    {
+      formatter.x86_64-linux = nixpkgs.legacyPackages.x86_64-linux.nixfmt-rfc-style;
+    };
+}
+```
+
 ## Development
 
+You are encouraged to test this out on your code and submit any undesirable formatting you find as an issue.
```

Comment:
@mightyiam, FYI

Reply:
I find that sentence a bit weird tbh. It feels hyper-specific and also a bit out of place, given that we also have CONTRIBUTING.md. Better to properly expand on our contribution flow in a separate PR instead of squeezing that in here

Reply:
#300